doc/api/url.md

```diff
@@ -305,6 +305,32 @@ to percent-encode may vary somewhat from what the [`url.parse()`][] and
 
 Gets and sets the port portion of the URL.
 
+The port value may be set as either a number or as a string containing a number
```

Comment:
Micro-nit, non-blocking: `may be set as either a number or as a string containing a number` -> `may be a number or a string containing a number`